packages/backend/src/core/transaction-count/ZksyncTransactionUpdater.ts

```diff
@@ -79,4 +81,13 @@ export class ZksyncTransactionUpdater {
 
     this.logger.info('Update enqueued')
   }
+
+  async getDailyTransactionCounts() {
+    const counts =
+      await this.zksyncTransactionRepository.getDailyTransactionCount()
+    return {
+      projectId: ProjectId('zksync'), // TODO: take from config
```

Comment:
@sz-piotr how do you suggest we solve this? Ideally we take it from a constant from config, but I'm open for suggestions.

Reply:
We could introduce this as a constant like this: `ProjectId.ZKSYNC`